demo_output/output.md

```
@@ -2,984 +1,1123 @@
+# List of Terminal Tools (A-Z) - Terminal Trove
 
-## March 11, 2025
+# List of Terminal Tools (A-Z)
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)